src/data/faq.json

```diff
@@ -542,20 +542,108 @@
+                                    "If you receive a corresponding message in the Corona-Warn-App, you can voluntarily request a new vaccination certificate or recovery certificate free of charge via the app. After you have given your consent, the app transmits the certificate or several certificates for verification to the RKI, which issues an updated certificate and transmits it to your app. You can then find the updated certificate in the Certificates tab.",
+                                    "<ul> <li>The old certificate is automatically moved to the trash and deleted after 30 days. The old certificate can be restored at any time within the 30 days.</li> <li>The possibility of reissue begins 28 days before the expiration of the certificate and ends 90 days after the expiration of the certificate</li> <li><strong>IMPORTANT</strong>: The reissue can be done a maximum of three times within this period. Because of this limitation, we recommend that you export the reissued certificate as a PDF file and print it out so that you can scan the QR code of the certificate again if necessary. (see &quot;<a href='#eu_dcc_export' target='_blank'>How do I create a printout of my EU Digital COVID Certificate?</a>&quot;</li> </ul>"
+                                ],
+                                "linkList": [
+                                    {
+                                        "title" : "Related FAQ articles:",
+                                        "links" : [
+                                            { "url" : "#dcc_replacement", "text": "Warum werden Zertifikate neu ausgestellt?"},
+                                            { "url" : "#dcc_replacement_which", "text": "Welche digitalen COVID Zertifikate der EU werden neu ausgestellt?"},
+                                            { "url" : "#dcc_replacement_vac_status", "text": "Verliere ich meinen Impfstatus wenn eines meiner Zertifikate abläuft?"},
+                                            { "url" : "#dcc_replacement_issues", "text": "I cannot renew a certificate in the app. What could be the reason?"},
+                                            { "url" : "#glossary_validity", "text": "Gültigkeit eines EU digitalen COVID-Zertifikats."}
+                                        ]
+                                    }
+                                ]
+                            },
+                            {
+                                "title": "Which EU digital COVID certificates are newly issued?",

```

Comment:
Am I the only one who finds the wording "newly issued" a bit confusing? Why not use "reissued"?